cli/src/logger/logzio.rs

```diff
@@ -0,0 +1,58 @@
+use chrono::Utc;
+use hyper::{client::Client, Body, Request};
+use log::{Level, Metadata, Record};
+use serde::{Deserialize, Serialize};
+use serde_json::json;
+use std::env;
+use tokio;
+
+#[derive(Deserialize, Serialize, Debug)]
+pub struct LogzIO;
+
+impl log::Log for LogzIO {
+  fn enabled(&self, metadata: &Metadata) -> bool {
+    metadata.level() <= Level::Info
+  }
+
+  fn log(&self, record: &Record) {
+    let utc_time = Utc::now().format("%FT%T%.3fZ");
+    let env = env::var("ALAN_TECH_ENV").unwrap_or("production".to_string());
+    let cluster = record.target(); // If no target defined default to `anycloud`
+    let token = match env.as_str() {
+      "local" => "ZERXpCvywsOBtNOXrqIzfpLiOnEXKXhb",
+      "staging" => "ZERXpCvywsOBtNOXrqIzfpLiOnEXKXhb",
+      "production" => "ZERXpCvywsOBtNOXrqIzfpLiOnEXKXhb",
+      _ => "",
```

Comment:
if all environments have the same token and that is another dimension/field in the log itself the we can just set it once? and if so, can we just make it a const at the top of the file?

Reply:
Right now all of them have the same, I did it this way to leave it prepare if we have multiple tokens.